src/pages/home.tsx

```diff
@@ -0,0 +1,35 @@
+import { useState } from "react";
+import NavBar from "~/components/ui/NavBar";
+
+const Home = () => {
+  const [page, setPage] = useState(1);
```

Comment:
Can refactor out magic literals
```suggestion
  const INITIAL_PAGE = 1;
  const [page, setPage] = useState(INITIAL_PAGE);
```